TeamCode/src/main/java/org/firstinspires/ftc/teamcode/teleop/MainTele.java

```diff
@@ -72,15 +72,23 @@ public void runOpMode() {
             // -----------------------------------------------------------------------------------------
             // NOTE: TO USE THESE FUNCTIONS, PRESS START B
             //OPERATOR FUNCTIONS
-
+            /*ADD CODE: FINE CONTROL ARM VIA RIGHT JOYSTICK Y, DROPPER VIA L/R BUMPER
+            * RESET DROPPER TO NEUTRAL, forwards, backwards ON R JOYSTICK PRESSED*/
+            /*
+            * Y: extend to high arm position, A: extend to low arm position
+            * B: duckspinner
+            * DPAD UP: autorun front intake, DPAD DOWN: autorun back intake
+            * left stick y: both intakes forwards/backwards, hold left stick for slow mode
+            * right stick y: change dropper position, up = back, dwn = fwds
+            * L/R BUMPERS: set servo pos*/
             // make sure you make the target negative
             boolean HIGH_EXTENDER = controller2.y == Btn.PRESSING;
             boolean LOW_EXTENDER = controller2.a == Btn.PRESSING;
 
             if(HIGH_EXTENDER) {
                 //im gonna hardcode the distance because heck you - simon
                 if(shouldGoToHigh) {
```

Comment:
this should probably have a moveExtenderToHighPosition funciton in robot or something like that- the logic shouldn't be here